serverless/src/autocomplete/handler.js

```diff
@@ -1,158 +1,10 @@
-import axios from 'axios'
-
-import { Client } from '@googlemaps/google-maps-services-js'
-
 import { buildParams } from '../util/cmr/buildParams'
 import { determineEarthdataEnvironment } from '../util/determineEarthdataEnvironment'
 import { doSearchRequest } from '../util/cmr/doSearchRequest'
-import { getApplicationConfig, getEnvironmentConfig } from '../../../sharedUtils/config'
-import { getGoogleMapsApiKey } from '../util/google/getGoogleMapsApiKey'
+import { getApplicationConfig } from '../../../sharedUtils/config'
 import { getJwtToken } from '../util/getJwtToken'
 import { parseError } from '../../../sharedUtils/parseError'
 
-/**
- * Search the Google Maps API endpoint
- * @param {query} query The spatial query provided from the user
- */
-const googleGeocode = async (query, earthdataEnvironment) => {
-  const client = new Client({})
-
-  // Retrieve the Google Maps API key
-  const apiKey = await getGoogleMapsApiKey(earthdataEnvironment)
-
-  const geocodeResult = await client
-    .geocode({
-      params: {
-        address: query,
-        key: apiKey
-      }
-    })
-
-  const { data } = geocodeResult
-  const { results } = data
-
-  const formattedResult = results.map((place) => {
-    const { geometry, formatted_address: formattedAddress } = place
-    const { bounds, location } = geometry
-
-    const { lat, lng } = location
-    const spatialResponse = {
-      point: [
-        lat, lng
-      ]
-    }
-
-    // If the results include a bounding box include it in the result
-    if (bounds) {
-      const { northeast, southwest } = bounds
-
-      spatialResponse.bounding_box = [
-        southwest.lng,
-        southwest.lat,
-        northeast.lng,
-        northeast.lat
-      ]
-    }
-
-    return {
-      name: formattedAddress,
-      ...spatialResponse
-    }
-  })
-
-  return formattedResult
-}
-
-/**
- * Search the Nominatim OpenStreetMaps API endpoint
- * @param {query} query The spatial query provided from the user
- */
-const nominatimGeocode = async (query) => {
```

Comment:
Double check that may also remove the `geocodingService: ${env:GEOCODING_SERVICE, 'nominatim'}` in `serverless.yaml`

Reply:
We can remove these but, we should also make sure as this propagates to remove the env vars from bamboo as well